contents/blog/changing-to-self-serve.md

```diff
@@ -0,0 +1,74 @@
+
+## What changed?
+
+Things went much better than we thought. For the last 6 months, our paid products have averaged 20% month over month revenue growth - that's an 8.9x increase per year (for context - [3x is considered great](https://twitter.com/nimaroohis/status/1525970172237991939?s=21&t=qzYbxjpPbHlPZ7Xjus_PUA), and [Lenny's Newsletter breaks this out in more detail](https://www.lennysnewsletter.com/p/what-is-a-good-growth-rate?s=r)). 
+
+Three things combined caused this:
+
+* we were able to get larger (generally, up to $70k) deals than we expected
+* we made it easier to _pay_ for our paid product, which [led to more deals](pricing-lessons) that we expected
+* we had more existing customer growth than we expected
+
+Then we noticed something else.
+
+Some customers are very fast (a couple of weeks, or less) to sell to, with minimal interaction, whereas others take months and are very intense. In fact, almost none of the customers that have put us through a big buying process have in fact converted to revenue yet.
+
+## What does Nail Self Serve mean?
+
+So we should do more of what's working.
+
+For product, this means:
+
+* Quality: Our core features (insights, recordings, feature-flags / experimentation) work like a Swiss watch
+* Self Service Subscription: Customers of every size sign-up, start using and subscribe to PostHog without asking us for help
+* Deployments: PostHog is the easiest self-hosted product to deploy and scale in the world
+
+For go to market, this means:
+
+* Customer Success: Focus on time to revenue (for example, one of our Customer Success team's key results is volume of deals in our target range per week - rather than trying to hit an overall revenue figure)
+* Marketing: Accelerate word of mouth growth
+* Website and Docs: Make community self serve (for example, [make our _docs_](https://squeak.posthog.com) rather than Slack the main support channel)
+
+## A snapshot of the view future
+
+Today, we're very good at serving large mid market B2B companies who can buy quickly (through self serving).
+
+I suspect we'll focus on B2C after the above is completed. From customer calls, it feels like this is how we'll go from landing $20k-$70k customers in a couple of weeks to $100k to $300k customers in the same, short time frame. It's much _more_ appealing to do this, than to sell to B2B Enterprises that'll take months or years to purchase from us.
+
+We already have plenty of (paying) customers in B2C, but to focus on it, would require stronger mobile support, greater scalability still and a suitable pricing model so we capture the right amount of value.
+
+Long term, we think the future of our revenue will come from large enterprise. We're a natural fit - data control, consolidating multiple vendors, and complete extensibility. However, we want to avoid _needing_ to go through complex purchasing decisions whilst there are easier ways to increase our numbers.
```

Comment:
```suggestion
Long term, we think the future of our revenue will come from large enterprise. We're a natural fit - data control, consolidating multiple vendors, and complete extensibility. However, we want to avoid _needing_ to go through complex purchasing decisions whilst there are easier ways to increase our numbers.

_Did you enjoy this article? Get more like it in your inbox every two weeks_

 <NewsletterForm
compact
/>
```